lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/exploring-mpg.adoc

```diff
@@ -0,0 +1,78 @@
+= Exploring the Fuel Efficiency Dataset
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+</style>
+++++
+For this page, you'll need to the  @starter-file{alg2-fuel-efficiency} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run", and evaluate `mpg-table` in the Interactions Area. Do you know what each column means?
+
+[cols="1a,1a", grid="none", frame="none", stripes="none"]
+|===
+|
+- *test* - the number of the fuel efficiency test
+- *model* - the car used in the test (A-F)
+|
+- *speed* - the speed at which the car was tested
+- *mpg* - the miles-per-gallon the car achieved at that speed
+|===
+
+@n Evaluate `A15`, `A45` and `A75` in the Interactions Area. What *model* of car is used in all three rows? @fitb{}{@ifsoln{`A`}}
+
+@n At what *speeds* is this model being tested in each of the rows? @fitb{}{@ifsoln{15, 55, and 75 miles per hour}}
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="1a,1a", options="frame",stripes="none"]
+!===
+| @n In Pyret, make a scatter plot showing the relationship between `speed` and `mpg`, using `model` as your labels, then sketch the resulting scatter plot below.
+|
+--
+Based on the scatter plot...
+
+@n What do you *Notice?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What do you *Wonder?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What speed is generally the most fuel-efficient? @fitb{}{@ifsoln{45-55mpg}}
+
+@n Which model is generally the most fuel-efficient? @fitb{}{@ifsoln{B}}
+
+@n Which model is generally the _least_ fuel-efficient? @fitb{}{@ifsoln{G}}
+--
+!===
+
+@n Do you see a correlation? Describe its form (linear/nonlinear/none), direction (positive/negative), and strength (strong/moderate/weak/none):
+
+@fitb{}{@ifsoln{There seems to be a somewhat strong, nonlinear correlation}}
+
+@fitb{}{@ifsoln{It looks like a parabola that opens down, with a dimple at the top}}
+
+@center{**Stop here! We'll continue after some discussion.**}
+
+@n A linear model is defined on line 22, with its Contract on line 21. What is its name? @fitb{2em}{@ifsoln{f}} Domain? @fitb{5em}{@ifsoln{Number}} Range? @fitb{}{@ifsoln{Number}}
+
+@n Fit the linear model `f` to the data using the `fit-model` function. What is the @math{R^2} value? @fitb{}{@ifsoln{-11.794}}.
```

Comment:
I couldn't remember the contract for `fit-model`. Doubt students will either.

Reply:
That's why they have their contracts pages! They'll have just used this function with the linear model lesson as well, so it should be relatively fresh

Reply:
I had also just done the linear model lesson... Want to remind them of the existence of the contracts page? Or do you think it's been utilized a lot in this pathway already? In my experience, not providing contracts means that students who are already resourceful / organized / feeling confident move through the material much faster than those who are already intimidated and take a while to find what they need, which then becomes a classroom management issue.

Reply:
I like the idea of reminding them about the contracts page! Done.